process_manager/templates/process_manager/index.html

```diff
@@ -1,18 +1,121 @@
 {% extends "main/base.html" %}
+{% load tz %}
 {% block title %}
   Home
 {% endblock title %}
 {% block extra_css %}
   <style>
     #hide-messages-button {
-      /* Certain themes invert the colour, so we invert back. */
       filter: invert(1);
       float: right;
     }
+    #show-messages-button {
+      display: none; /* Hide on load */
+    }
     #message-list {
       max-height: 80vh;
       overflow-y: auto;
     }
+    .table-container {
```

Comment:
There is a lot of extra css in this PR, and it's all put into the index page for the process_manager. Would it be better off that it is moved to the `base.html` so all the pages (in both the process_manager and the controller) inherit the same css adjustments?